main.tf

```diff
@@ -140,6 +140,7 @@ resource "aws_cloudformation_stack" "datadog_forwarder" {
     DdApiKey          = "this_value_is_not_used"
     DdApiKeySecretArn = aws_secretsmanager_secret.api_key.0.arn
     DdSite            = var.site_url
+    DdTags            = var.datadog_tags
```

Comment:
DdTags must be a string but the datadog_tags variable is a list(string). Can you transform this variable into a string?